docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/DataViewEnumerable.tt

```diff
@@ -3,29 +3,39 @@
 string NameSpace = "Samples.Dynamic";
 string ClassName="DataViewEnumerable";
 string AddExtraClass = "true";
-string ExampleShortDoc = @"// A simple case of creating IDataView from IEnumerable.";
-string Example = @"// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+string ExampleShortDoc = @"// A simple case of creating IDataView from
+        //IEnumerable.";
+string Example = @"// Create a new context for ML.NET operations. It can be used for
+		    // exception tracking and logging,
             // as a catalog of available operations and as the source of randomness.
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable.
-            IEnumerable<SampleTemperatureData> enumerableOfData = GetSampleTemperatureData(5);
+            IEnumerable<SampleTemperatureData> enumerableOfData =
+			    GetSampleTemperatureData(5);
 
             // Load dataset into an IDataView. 
             IDataView data = mlContext.Data.LoadFromEnumerable(enumerableOfData);
 
-            // We can now examine the records in the IDataView. We first create an enumerable of rows in the IDataView.
-            var rowEnumerable = mlContext.Data.CreateEnumerable<SampleTemperatureData>(data, reuseRowObject: true);
-
-            // SampleTemperatureDataWithLatitude has the definition of a Latitude column of type float. 
-            // We can use the parameter ignoreMissingColumns to true to ignore any missing columns in the IDataView.
-            // The produced enumerable will have the Latitude field set to the default for the data type, in this case 0. 
-            var rowEnumerableIgnoreMissing = mlContext.Data.CreateEnumerable<SampleTemperatureDataWithLatitude>(data,
-                reuseRowObject: true, ignoreMissingColumns: true);
+            // We can now examine the records in the IDataView. We first create an
+			// enumerable of rows in the IDataView.
```

Comment:
Extra whitespace

Reply:
Should be fixed. Some tt files have certain indentations that insure that the comments in the cs files align.